File: mixedProblems/problem_7.py
# Problem 7: How do you calculate the number of vowels and consonants in a String?

myString = input("Enter your string: ")
lowerString = myString.lower()

vowelCount = 0
consonantCount = 0
for word in lowerString:
    if word=='a' or word =='e' or word=='i' or word=='o' or word=='u':
        vowelCount+=1
    else:
        consonantCount+=1

print(f'Total vowel: {vowelCount} and Total consonant: {consonantCount}')

File: mixedProblems/problem_7.js
// Problem 7: How do you calculate the number of vowels and consonants in a String?

function myFunc(word){
    var ArrayOfString  = word.toLowerCase().split('')
    var vawel = 0
    var cons =0
    for(var i =0;i<ArrayOfString.length;i++){
        if(ArrayOfString[i]=='a' || ArrayOfString[i]=='e' || ArrayOfString[i]=='i' || ArrayOfString[i]=='o' || ArrayOfString[i]=='u'){
            vawel++
        }else{
            cons++
        }
    }
    console.log(`Number of Vowel is ${vawel} and Number of Consonent ${cons}`)
}

myFunc("Howareyou")